python/django_bolt/testing/dbshare.py

```
@@ -0,0 +1,251 @@
+    """Store that gives all threads the shared connections, and no others.
+
+    ``ConnectionHandler`` reads and writes connections as attributes of
+    ``_connections``, which is normally an ``asgiref`` Local with
+    ``thread_critical=True``. Shared aliases must resolve to one object for
+    every thread. Every other alias must stay thread-local: a connection that
+    Django did not mark as shareable raises ``DatabaseError`` when a second
+    thread uses it, thus an alias this client did not share is handed back to
+    the original store.
+    """
+
+    def __init__(self, shared: dict[str, Any], fallback: Any) -> None:
```

Ruff (0.16.1)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `fallback`

(ANN401)